docs/source/conf.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python3
```

Comment:
It is not clear to me who runs this script ...

Reply:
The script is run by Sphinx. If you run the .sh script it will run Sphinx which runs this.

Reply:
I added a comment at the start of conf.py to this effect.